en/concept/control_allocation.md

```diff
@@ -0,0 +1,70 @@
+# Control Allocation (Mixing)
+
+The PX4 architecture ensures that the airframe layout does not require special case handling in the core controllers.
+
+Mixing means to take force commands (e.g. `turn right`) and translate them to actuator commands which control motors or servos.
+For a plane with one servo per aileron this means to command one of them high and the other low.
+The same applies for multicopters: pitching forward requires changing the speed of all motors.
+
+Separating the mixer logic from the actual attitude/rate controller greatly improves reusability.
```

Comment:
@bkueng This is a little confusing to me (I understand it, but find it hard to read). I have attempted to re-write and further capture that we also abstract the output function mapping.

```suggestion
PX4 takes desired torque and thrust commands from the core controllers and translates them to actuator commands which control motors or servos.

The translation depends on the physical geometry of the airframe.
For example, given a torque command to "turn right" (say):
- A plane with one servo per aileron will command one of servo high and the other low.
- A multicopter will yaw right by changing the speed of all motors.

PX4 separates this translation logic, which is referred to as "mixing" from the attitude/rate controller.
This ensures that the core controllers do not require special handling for each airframe geometry, and greatly improves reusability.

In addition, PX4 abstracts the mapping of output functions to specific hardware outputs.
This means that any motor or servo can be assigned to almost any controller output.  
```

Reply:
This is copied from the existing page. Yes yours is better.

`can be assigned to almost any controller output` -> `can be assigned to almost any physical output`?
`The whole process is known as "control allocation"` -> I guess it can be interpreted in different ways, we typically use it as synonym for "mixing" (so w/o the physical output assignment). But then again we also use it to refer to the whole new architecture, which technically isn't accurate.

Reply:
OK, I have removed that last point "The whole process is known as "control allocation". It just seems odd to have two terms - then define mixing but not "control allocation".

Also took your first suggestion.